PROVESFlightControllerReference/test/int/mode_manager_test.py

```
@@ -594,8 +596,13 @@ def test_safe_09_command_loss_triggers_safe_mode_and_reboot(
     )
 
     # stopWatchdog was called after safe mode entry — hardware reset expected in ~30 seconds
-    logger.info("Waiting for hardware reboot triggered by watchdog stop (~60s)...")
-    time.sleep(60.0)
+    reboot_start: TimeType = TimeType().set_datetime(
+        datetime.now(), time_base=TimeType.TimeBase("TB_DONT_CARE")
```

Ruff (0.15.21)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)